nbs/15_gsa_gls/30-real_phenotypes/README.md

```diff
@@ -0,0 +1,169 @@
+# Overview
+
+This folder has the scripts to run GLS PhenoPLIER (associations between LVs/gene modules and traits on randomly generated phenotypes (`../15_spredixcan`).
+
+Before running these steps, **it is necessary** to generate a correlation matrix for predicted gene expression _specific_ for these random phenotypes (see `nbs/15_gsa_gls/README.md`).
+
+
+## Setup
+
+### Penn's LPC cluster
+
+Load Penn's LPC-specific paths and PhenoPLIER configuration.
+Change paths accordingly.
+
+```bash
+# load conda environment
+module load miniconda/3
+conda activate ~/software/conda_envs/phenoplier/
+
+# load LPC-specific paths
+. ~/projects/phenoplier/scripts/pmacs_penn/env.sh
+
+# set the executor of commands to "bsub" (to submit the jobs)
+export PHENOPLIER_JOBS_EXECUTOR="bsub"
+
+# load in bash session all PhenoPLIER environmental variables
+eval `python ~/projects/phenoplier/libs/conf.py`
+
+# make sure they were loaded correctly
+# should output something like /project/...
+echo $PHENOPLIER_ROOT_DIR
+```
+
+### Desktop computer
```

Comment:
is it possible to run this on a desktop computer? just curious

Reply:
Yes, it is! Sorry for the delay in replying, I thought I responded to all your comments!